src/foldermodel.cpp

```diff
@@ -223,6 +223,18 @@ QVariant FolderModel::data(const QModelIndex & index, int role/* = Qt::DisplayRo
           const char* name = fm_file_info_get_disp_owner(info);
           return QString::fromUtf8(name);
         }
+        case ColumnFileID3Title: {
+          ID3DataModel id3Data (fm_path_to_str(fm_file_info_get_path(info)));
```

Comment:
The strings from `fm_path_to_str()` should to be released with `g_free()` later (or it will be a memory leak).